app/src/test/java/edu/uc/hornunjb/weatherwear/WeatherResponseUnitTest.kt

```diff
@@ -0,0 +1,23 @@
+package edu.uc.hornunjb.weatherwear
+
+import edu.uc.hornunjb.weatherwear.dto.*
+import org.junit.Test
+
+import org.junit.Assert.*
+
+/**
+ * Unit Test for the WeatherResponse object
+ *
+ * See [testing documentation](http://d.android.com/tools/testing).
+ */
+class WeatherResponseUnitTest {
+    @Test
+    fun confirmSunny_outputsSpaghetti() {
+        var weatherResponse = WeatherResponse(Coord(), ArrayList<Weather>(), Main(), Wind(), Sys(), "Cincinnati")
+        var weatherObject = Weather()
+        weatherObject.description = "sunny"
+        weatherResponse.weather.add(weatherObject)
+
+        assertEquals("It is sunny in Cincinnati", weatherResponse.toString())
```

Comment:
I wrote a really simple unit test to just ensure that your object instantiation for the WeatherResponse class works as intended. More unit tests should probably be written to test actual functionality, but this should act as a bit of a head start. (this is one of those situations I stated earlier where having a descriptive toString() method is beneficial :) )

Reply:
Excellent.  Increase test functions.